cmd/geth/dbcmd.go

```diff
@@ -289,6 +301,67 @@ func inspect(ctx *cli.Context) error {
 	return rawdb.InspectDatabase(db, prefix, start)
 }
 
+func checkStateContent(ctx *cli.Context) error {
+	var (
+		prefix []byte
+		start  []byte
+	)
+	if ctx.NArg() > 2 {
```

Comment:
Does it make sense to have prefix? I know we can still specify a prefix with `123` so that only the nodes with prefix 123 will be checked. But the node prefix doesn't have too much meaning from my perspective. Since it's just random hash.

Perhaps we can change the prefix to limit. Then we can iterate the nodes in  key space [start, limit].

Reply:
I agree it doesn't make sense to have a prefix. I don't see a need for 'limit' either. If you stop for some reason (e.g. I did the other day, when I had forgotten to put it in a `tmux`), then it's nice to `start` again where you left off, but I don't see any reason to specify where you want to end.